docs/units/equivalencies.rst

```diff
@@ -111,8 +111,7 @@ energy can be converted.
 These equivalencies even work with non-base units::
 
   >>> # Inches to calories
-  >>> from astropy.units import imperial
-  >>> imperial.inch.to(imperial.Cal, equivalencies=u.spectral())  # doctest: +FLOAT_CMP
+  >>> u.imperial.inch.to(u.imperial.Cal, equivalencies=u.spectral())  # doctest: +FLOAT_CMP
```

Comment:
Yea, it's technically unnecessary (just a convention) but it eliminates the need for an extra `u.`. I would leave as is, but don't have a strong opinion so, fine.